crates/compiler/mono/src/ir.rs

```diff
@@ -2463,17 +2419,6 @@ fn from_can_let<'a>(
 
                 lower_rest!(variable, cont.value)
             }
-            Accessor(accessor_data) => {
```

Comment:
wait where did this guy go?

Reply:
Now it's compiled as if it was an alias to a proc. We currently compile

```
ra = .field
```

as

```
ra = \#rcd -[__ra]-> #rcd.field
```

anyway, so the existing association  incorrect. By deleting this block, we end up compiling this as

```
__ra1 = \#rcd -[__ra1] -> #rcd.field

ra = LambdaSet { __ra1 }
```

which is what we want, whereas before what was happening was an association of the proc `ra` with `\#rcd -[__ra]-> #rcd.field`, but this is wrong because the lambda sets don't line up.